TROUBLESHOOTING.md

```diff
@@ -18,9 +18,10 @@ This configuration allows Ollama to accept connections from any source.
 
 Ensure that the Ollama URL is correctly formatted in the application settings. Follow these steps:
 
+- If your Ollama runs in a different host than Web UI make sure Ollama host address is provided when running Web UI container via `OLLAMA_API_BASE_URL` environment variable. [(e.g. OLLAMA_API_BASE_URL=http://192.168.1.1/api)](https://github.com/ollama-webui/ollama-webui#accessing-external-ollama-on-a-different-server)
```

Comment:
Thanks for the PR! I believe it would be better to have `OLLAMA_API_BASE_URL=http://192.168.1.1:11434/api` as an example. Otherwise, LGTM!